grammar.js

```diff
@@ -953,30 +955,21 @@ module.exports = grammar({
     )),
 
     // http://stackoverflow.com/questions/13014947/regex-to-match-a-c-style-multiline-comment/36328890#36328890
-    comment: _ => token(choice(
-      seq('//', /.*/),
-      seq(
-        '/*',
-        /[^*]*\*+([^/*][^*]*\*+)*/,
-        '/',
-      ),
-      // https://tc39.es/ecma262/#sec-html-like-comments
-      seq('<!--', /.*/),
```

Comment:
what about those comments? We don't support them anymore? They were deprecated?

Reply:
I guess it was moved [here](https://github.com/tree-sitter/tree-sitter-javascript/pull/288/files#diff-bbcf6c785b007d9655b443abc204cc3cc32aa03cea190cb865fe44252dc6f1dfR223)

Reply:
Yeah it was moved to the external scanner as its own `html_comment` node because it was interfering with this code:
```
if(n-->0){}
```